tangelo/toolboxes/operators/operators.py

```diff
@@ -20,17 +20,102 @@
 from collections import OrderedDict
 
 import numpy as np
+from openfermion.ops.operators.symbolic_operator import COEFFICIENT_TYPES
```

Comment:
This is currently not used and Pycharm should be telling you that (code is gray)